samples/server/go-server/api/server/storage/pgsqlstore.go

```diff
@@ -126,22 +130,36 @@ func (pg *pgSQLStore) GetProject(pID string) (*pb.Project, error) {
 	return &pb.Project{Name: pName}, nil
 }
 
-// ListProjects returns the project id for all projects from the store
-func (pg *pgSQLStore) ListProjects(filters string) ([]*pb.Project, error) {
-	rows, err := pg.DB.Query(listProjects)
+var paginationKey = "XxoPtCUzrUv4JV5dS+yQ+MdW7yLEJnRMwigVY/bpgtQ="
```

Comment:
Is this meant to be here, shouldn't this be read from the config?

Reply:
You are right. Forgot to remove this. Fixed in 5108566.